website/docs/feature-flag-tutorials/use-cases/a-b-testing.md

```diff
@@ -1,112 +1,239 @@
+There are two more advanced extensions of a default strategy that you will see available to customize in the form:
+
+-   [Strategy variants](/reference/strategy-variants)
+-   [Strategy constraints](/reference/strategy-constraints)
+
+With strategy variants and constraints, you can extend your overall strategy. They help you define more granular conditions for your feature beyond the rollout percentage. We recommend using strategy variants to configure an A/B test.
 
-For A/B testing, _strategy variants_ are most applicable for more granular conditions of a feature release. In the next section, we’ll explore how to apply a strategy variant on top of an A/B test for more advanced use cases.
+[Strategy variants](/reference/strategy-variants) let you expose a particular version of a feature to select user bases when a flag is enabled. You can then collect data to determine which variant performs better, which we'll cover later in this tutorial.
 
-### Applying Strategy Variants
+Using strategy variants in your activation strategy is the canonical way to run A/B tests with Unleash and your application.
 
-Using strategy variants in your activation strategy is the canonical way to run A/B tests with Unleash and your application. You can expose a particular version of the feature to select user bases when a flag is enabled. From there, a way to use the variants is to view the performance metrics and see which is more efficient.
+![This diagram breaks down how strategy variants sit on top activation strategies for flags in Unleash.](/img/tutorial-building-blocks-strategy-variants.png)
 
 A variant has four components that define it:
 
--   **name**: This must be unique among the strategy's variants. When working with a feature with variants in a client, you will typically use the variant's name to find out which variant it is.
--   **weight**: The weight is the likelihood of any one user getting this specific variant. See the weights section for more info.
--   **value**
--   **(optional) payload**: A variant can also have an associated payload. Use this to deliver more data or context. See the payload section for more details.
+-   a name: This must be unique among the strategy's variants. You typically use the name to identify the variant in your client.
+-   a weight: The [variant weight](/reference/strategy-variants#variant-weight) is the likelihood of any one user getting this specific variant.
+-   an optional payload: A variant can also have an associated [payload](/reference/strategy-variants#variant-payload) to deliver more data or context.
+-   a value: This describes the purpose of the variant. For example, "Most Read" or "Most Popular".
```

Comment:
I don't think this is correct. Also, the value is not really the property of the variant, but it's a property of the payload. so the optional payload has a type and value

Reply:
the value is the actual payload

Reply:
So this would be an example:

payload type: json
payload value: 
```
{
"color": "blue",
"position: 1
}
```

Reply:
I see! It's the _entire_ payload. It seemed like a parameter of the payload not the whole thing, based on the Unleash admin UI

Reply:
Yes the payload 'entity' has a type, and a value, where the value is the actual payload you want to be sent back in the response. But at the moment the above docs say that value or a property of the variant